server/server.go

```diff
@@ -46,6 +47,10 @@ func NewSimulatorServer(cfg *config.Config, dic *di.Container) *SimulatorServer
 
 	v1.GET("/export", exportHandler.Export)
 	v1.POST("/import", exportHandler.Import)
+
+	// server push
+	v1.GET("/watchresources", watcherHandler.WatchResources)
```

Comment:
Add this to https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/docs/api.md